File: src/content/cre/code-snippets/index-trigger-callback-example.ts
cre.handler(
  cronTrigger.trigger({ schedule: "0 */10 * * * *" }), // trigger fires every 10 minutes
  onCronTrigger // your callback function
)

function onCronTrigger(runtime: Runtime<Config>): Record<string, never> {
  // Create SDK clients and call capabilities
  return {}
}

File: src/content/cre/code-snippets/index-trigger-callback-example.go
cre.Handler(
  cron.Trigger(&cron.Config{Schedule: "0 */10 * * * *"}), // Trigger fires every 10 minutes
  onCronTrigger, // your Go callback
)

func onCronTrigger(config *Config, runtime cre.Runtime, trigger *cron.Payload) (struct{}, error) {
  // Create SDK clients and call capabilities
  return struct{}{}, nil
}